src/check_links.ts

```diff
@@ -0,0 +1,15 @@
+function isNpmLink(input: string): boolean {
+
+    const pattern = /^https:\/\/www\.npmjs\.com\/package\/[a-zA-Z]+$/;
```

Comment:
could you make it so that the regex also accepts when URL doesn't start with https:www?

Like when a "npmjs.com/package/safe-regex" is passed in, it will also acccept.